src/store/reducers/sidebar.js

```diff
@@ -0,0 +1,13 @@
+import { CHANGE_SIDEBAR_INDEX } from '../constants/constants';
+
+const slideBarReducer = (state = 0, { type, payload = 0 }) => {
+  switch (type) {
+  case CHANGE_SIDEBAR_INDEX:
+    return payload;
```

Comment:
Let's destructure the payload in all the cases. It is going to be easier to read what's changing every time, also we keep sure we are updating a single prop every time.

Reply:
Destructure? 🤔

Reply:
<img width="687" alt="Screen Shot 2020-12-16 at 18 41 58" src="https://user-images.githubusercontent.com/3811355/102427348-62e63900-3fce-11eb-8448-5d7bf58c299b.png">

Destructure the payload, or assign the correct prop instead of just updating the whole state.